modules/Settings/Menu/actions/SaveAjax.php

```diff
@@ -62,7 +62,7 @@ public function removeMenu(\App\Request $request)
 	public function updateSequence(\App\Request $request)
 	{
 		$recordModel = Settings_Menu_Record_Model::getCleanInstance();
-		$recordModel->saveSequence($request->get('mdata'), true);
+		$recordModel->saveSequence($request->getArray('mdata', 'Text'), true);
```

Comment:
getMultiDimensionArray

Reply:
Can not be "getMultiDimensionArray" because of nesting.